### PR TITLE
fix fsdriver router topic name

### PR DESCRIPTION
**Pull request by bendavies**
As discussed in gitter.

### CODE REVIEW

File: pkg/enqueue/Client/Config.php [hunk 1]
```diff
@@ -154,4 +154,14 @@ public static function create(
             $routerProcessorName ?: 'router'
         );
     }
+
+    /**
+     * @param string $name
+     *
+     * @return string
+     */
+    private function getQueueName($name)
```
Comment (makasim) [inline pkg/enqueue/Client/Config.php]: Could you please rename it to `getDestinationName`

Comment (bendavies) [inline pkg/enqueue/Client/Config.php]: done

**Comment by makasim** [comment]
I have just realized that the router topic must not contain an app it.

The router topic is a shared one. Several applications may send messages to it and hence talk to each other. It is a ground part of message bus support. 

And since message bus is not supported by `fs` transport we should isolate the change in FsDriver. When a sendToRouter method is called we should instead of a topic create a queue and use it.

**Comment by bendavies** [comment]
no problem, i'll take a look.

**Comment by makasim** [comment]
@bendavies yeap that looks good.

**Comment by bendavies** [comment]
@makasim i'll add a test to cover this case.

**Comment by bendavies** [comment]
@makasim done

**Comment by makasim** [comment]
Thanks!

**Pull request merged**